equinix/resource_fabric_service_profile.go

```diff
@@ -880,49 +880,42 @@ func customFieldFabricSpToTerra(customFieldl []v4.CustomField) []map[string]inte
 	return mappedCustomFieldl
 }
 
-func processStepFabricSpToTerra(processStepl []v4.ProcessStep) []map[string]interface{} {
+func processStepFabricSpToTerra(processStepl []v4.ProcessStep) []interface{} {
```

Comment:
If the method input is a slice, then we need to keep the for loop. We will add `map[string]interface{}` elements to a `[]interface{}` made with the right length by index. Then return the `[]interface{}` directly. We won't need to use `append()`.

Reply:
I know during our session I may have started this one, but was just trying to represent the concept and missed the input being a list type.